docs/compendium/substitute.md

```diff
@@ -133,20 +140,86 @@ Result will be a list of compendia ids that have been substituted
 
 ## List related substituted Compendia
```

Comment:
Merge this with `## List substituted Compendia`, the filters are optional. There should be only one `##` level section for `.../substitutions`.